services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaServiceITest.kt

```diff
@@ -148,6 +150,52 @@ class HankeKayttajaServiceITest : IntegrationTest() {
         }
     }
 
+    @Nested
+    inner class HasPermission {
+        private fun kayttooikeustaso(hasEditPermission: Boolean) =
+            when (hasEditPermission) {
+                true -> Kayttooikeustaso.HANKEMUOKKAUS
+                false -> Kayttooikeustaso.KATSELUOIKEUS
+            }
+
+        @ParameterizedTest
+        @ValueSource(booleans = [true, false])
+        fun `returns the correct answer when the kayttaja is identified`(expected: Boolean) {
```

Comment:
Parameter name `expected` was not quite clear to me at fisrt. Could `hasEditPermission` be used here as well as in `kayttooikeustaso` function above?

Reply:
Yes, `hasEditPermission` is better. Thank you.